sections/shared/Layout/Banner/Banner.tsx

```diff
@@ -1,20 +1,31 @@
 import { FC, useCallback, useEffect, useState } from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { Svg } from 'react-optimized-image';
 import Color from 'color';
 
 import { FlexDivCentered } from 'styles/common';
 import CloseIcon from 'assets/svg/app/close.svg';
 
+export enum BannerType {
+	INFORMATION = 'information',
+	ATTENTION = 'attention',
+	WARNING = 'warning',
+}
+
 type BannerProps = {
 	message: JSX.Element;
-	localStorageKey: string;
+	localStorageKey?: string | undefined;
+	type?: BannerType;
 };
 
-const Banner: FC<BannerProps> = ({ message, localStorageKey }) => {
+const Banner: FC<BannerProps> = ({ message, localStorageKey, type = BannerType.INFORMATION }) => {
 	const [isBannerVisible, setIsBannerVisible] = useState<boolean>(true);
 
 	const fetchFromLocalStorage = useCallback(() => {
+		if (!localStorageKey) {
+			setIsBannerVisible(true);
```

Comment:
is this meant to be ```false``` since localStorageKey determines which banner to show?

Reply:
`WARNING` can't be dismissed. That's something I decided. I don't want people to hide their liquidation warnings and then forget about it.

So if there is no storage key, then you assume you can't hide the banner.